#! /bin/sh /usr/share/dpatch/dpatch-run
## 06_error_newline.dpatch by Jonas Meurer <jonas@freesources.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: add missing newline to "Command failed."

@DPATCH@

--- cryptsetup-1.0.3/src/cryptsetup.c
+++ cryptsetup-1.0.3/src/cryptsetup.c
@@ -91,7 +91,7 @@
 	if (*error)
 		fprintf(stderr, ": %s\n", error);
 	else
-		fputs(".", stderr);
+		fputs(".\n", stderr);
 	return;
 }
 
